### PR TITLE
Compatibility issues fixed

### PR DESCRIPTION
**Pull request by elviramartinv**
This PR aims to update the ntuple producer for HHBtag training

### CODE REVIEW

File: Studies/HHBTag/CreateTrainingSkim.py [hunk 1]
```diff
@@ -27,30 +33,38 @@ def createSkim(inFile, outFile, period, sample, X_mass, mpv, snapshotOptions):
...
 
-    df = df.Define('genChannel', 'genHttCand.channel()')
-    df = df.Define('recoChannel', 'httCand.channel()')
+    df = df.Define('genChannel', 'genHttCandidate->channel()')
+    df = df.Define('recoChannel', 'HttCandidate.channel()')
 
     df = df.Filter("genChannel == recoChannel", "SameGenRecoChannels")
-    df = df.Filter("GenRecoMatching(genHttCand, httCand, 0.2)", "SameGenRecoHTT")
-    df = Baseline.RequestOnlyResolvedRecoJets(df)
+    # df = df.Filter("GenRecoMatching(genHttCandidate, HttCandidate, 0.2)", "SameGenRecoHTT")
```
Comment (kandrosov) [inline Studies/HHBTag/CreateTrainingSkim.py:44]: I think is better to require that genHttCandidate matches with the selected httCandidate

**Review by kandrosov — CHANGES_REQUESTED** [review]

File: Studies/HHBTag/CreateTrainingSkim.py (hunk 1 above)
Comment (elviramartinv) [reply Studies/HHBTag/CreateTrainingSkim.py:44]: But GenRecoMatching is a function that is no longer defined in the BaselineSelection.py (when I started looking at this, it was indeed defined). I'm not sure if I should require the matching in some way?

Comment (kandrosov) [reply Studies/HHBTag/CreateTrainingSkim.py:44]: It is still [there](https://github.com/cms-hh-bbtautau/Framework/blob/main/Common/BaselineRecoSelection.h#L124), but is commented for some reason: likely some compilation errors... Could you please fix it?

**Comment by elviramartinv** [comment]
"node_index" is already set as an input argument instead of defining it within createtrainingskim.py, and the lines of the genjet variables that I used for my studies have been commented out.

**Review by kandrosov — APPROVED** [review]

**Comment by kandrosov** [comment]
@elviramartinv thank you! Please resolve conflicts so that I can merge this PR.

**Pull request merged**